contribute/design/release-management.md

```diff
@@ -0,0 +1,33 @@
+# OpenEBS Release Management
+
+OpenEBS Components are released as container images with versioned tags. The release process is triggered by creating a Release Tag on the repository. For major releases, the Release Tag is created on the master branch and for minor release, the release tag is triggered on the corresponding release branch.
+
+The release process involves the following stages:
+- Release Candidate Builds
+- Update Installer and Documentation
+- Update the charts like Helm stable and other partner charts. (Rancher, OpenShift, IBM ICP Communty Charts, Netapp NKS Trusted Charts (formerly StackPointCloud), AWS Marketplace)
+- Update the openebs-operator.yaml
+- Final Release
+
+Release Candidate Builds
+On reaching the feature freeze date, the repositories are tagged with "Release-Name-RC1" tag. (Example 1.0.0-RC1).
+The release tags are applied in the following order:
+- openebs/libstor
+- openebs/cstor
+- openebs/istgt
+- openebs/jiva
+- openebs/ndm
+- openebs/external-storage
+- openebs/velero-plugin
+- openebs/maya
+
+The e2e pipeline and exploratory tests are executed using the RC tagged images. Any issues identified during this RC testing are tagged as release blockers.
+After the issues are fixed and verified by the CI, next RC is triggered.(Example: 0.1.0-RC2)
```

Comment:
n_ : to follow the release example above - make it 1.0.0-RC2

Reply:
Done.